doctor/docs/base.py

```diff
@@ -295,6 +363,19 @@ def prefix_lines(lines, prefix):
     return [prefix + line for line in lines]
 
 
+def class_name_to_resource_name(class_name: str) -> str:
+    """Converts a camel case class name to a resource name with spaces.
+
+    >>> class_name_to_resource_name('FooBarObject')
+    'Foo Bar Object'
+
+    :param class_name: The name to convert.
+    :returns: The resource name.
+    """
+    s = re.sub('(.)([A-Z][a-z]+)', r'\1 \2', class_name)
```

Comment:
What case are you expecting this substitution to handle? It looks like the substitution below would handle things, and when I ran the examples from your test case through that, all of them were converted correctly.

Reply:
I forgot to add a test for this case.  This ensures if the class names starts with an acronym in all caps that it splits properly.

e.g. 

```python
>>> class_name_to_resource_name('HTTPResponse')
'HTTP Response'
```